src/config/dune

```
@@ -0,0 +1,3 @@
+(executable
+ (name discover)
+ (libraries dune.configurator))
```

AFAIK `dune.configure` was deprecated a few years ago
```suggestion
 (libraries dune-configurator))
```

---

@kit-ty-kate are you sure?
```
ℤ dune build
File "src/config/dune", line 3, characters 12-29:
3 |  (libraries dune-configurator))
                ^^^^^^^^^^^^^^^^^
Error: Library "dune-configurator" not found.
-> required by _build/default/src/config/discover.exe
-> required by alias src/config/all
-> required by alias default
```